applications/swimming_DEM_application/custom_utilities/binbased_DEM_fluid_coupled_mapping.cpp

```diff
@@ -1225,7 +1225,12 @@ void BinBasedDEMFluidCoupledMapping<TDim, TBaseTypeOfSwimmingParticle>::Calculat
             double& fluid_fraction = i_node->FastGetSolutionStepValue(FLUID_FRACTION);
 
             if (mCouplingType != 4){
-                fluid_fraction = 1.0 - fluid_fraction / i_node->FastGetSolutionStepValue(NODAL_AREA);
+		double nodalFluidVolume=i_node->FastGetSolutionStepValue(NODAL_AREA);
```

Comment:
missing & ? otherwise we are not modifying `i_node->FastGetSolutionStepValue(NODAL_AREA)`

Reply:
nothing